watertap/unit_models/electrodialysis_0d.py

```diff
@@ -0,0 +1,938 @@
+    **PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+
+    CONFIG.declare(
+        "property_package_args",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing property packages",
+            doc="""A ConfigBlock with arguments to be passed to a property block(s)
+    and used when constructing these,
+    **default** - None.
+    **Valid values:** {
+    see property package for documentation.}""",
+        ),
+    )
+
+    def build(self):
+        # build always starts by calling super().build()
+        # This triggers a lot of boilerplate in the background for you
+        super().build()
+        # this creates blank scaling factors, which are populated later
+        self.scaling_factor = Suffix(direction=Suffix.EXPORT)
+
+        # Next, get the base units of measurement from the property definition
+
+        # Create essential sets.
+        self.membrane_set = Set(initialize=["cem", "aem"])
+
+        # Create unit model parameters and vars
+        self.water_density = Param(
+            initialize=1000,
+            mutable=False,
+            units=pyunits.kg * pyunits.m**-3,
+            doc="density of water",
+        )
+        self.water_MW = Param(
+            initialize=18.015e-3,
+            mutable=False,
+            units=pyunits.kg * pyunits.mole**-1,
+            doc="molecular weight of water",
+        )
+        self.cell_pair_num = Var(
+            initialize=1,
+            domain=NonNegativeIntegers,
+            bounds=(1, Inf),
+            units=pyunits.dimensionless,
+            doc="cell pair number in a stack",
+        )
+
+        # electrodialysis cell dimensional properties
+        self.cell_width = Var(
+            initialize=0.1,
+            bounds=(1e-3, 1e2),
+            units=pyunits.meter,
+            doc="The width of the electrodialysis cell, denoted as b in the model description",
+        )
+        self.cell_length = Var(
+            initialize=0.5,
+            bounds=(1e-3, 1e2),
+            units=pyunits.meter,
+            doc="The length of the electrodialysis cell, denoted as l in the model description",
+        )
+        self.spacer_thickness = Var(
+            initialize=0.0001,
+            units=pyunits.meter,
+            doc="The distance between the concecutive aem and cem",
+        )
+
+        # Material and Operational properties
+        self.membrane_thickness = Var(
+            self.membrane_set,
+            initialize=0.0001,
+            bounds=(1e-6, 1e-1),
+            units=pyunits.meter,
+            doc="Membrane thickness",
+        )
+        self.solute_diffusivity_membrane = Var(
+            self.membrane_set,
+            self.config.property_package.ion_set
+            | self.config.property_package.solute_set,
+            initialize=1e-10,
+            # bounds=(1e-11, 1e-6),
```

Comment:
```suggestion
            bounds=(1e-11, 1e-6),
```
Why is this being commented out? Is the lower bound low enough for this variable?

Reply:
I put it back.  It should be a temporary disabling during my debug.  It does not cause a problem till now.